src/stories/5-custom/iframe/iframe.stories.tsx

```diff
@@ -0,0 +1,136 @@
+			table: {
+				type: {
+					summary: "string",
+				},
+				defaultValue: { summary: null },
+			},
+		},
+	},
+};
+export default meta;
+
+const host = `${window.location.protocol}//${window.location.host}`;
+
+export const Default = DefaultStoryTemplate<IIframeSchema>("iframe-default").bind({});
+Default.args = {
+	referenceKey: "iframe",
+	validationTimeout: -1,
+	src: `${host}/iframe.html?viewMode=story&id=custom-iframe--default-child`,
+};
+export const DefaultChild: StoryObj = {
+	render: () => <ChildDefault />,
+	tags: ["!autodocs", "!dev"],
+};
+
+export const Validation = DefaultStoryTemplate<IIframeSchema>("iframe-validation").bind({});
+Validation.args = {
+	referenceKey: "iframe",
+	src: `${host}/iframe.html?viewMode=story&id=custom-iframe--validation-child`,
+};
+Validation.tags = ["!autodocs"];
+export const ValidationChild: StoryObj = {
+	render: () => <ChildValidation />,
+	tags: ["!autodocs", "!dev"],
+};
+
+export const Resize = DefaultStoryTemplate<IIframeSchema>("iframe-resize", true).bind({});
+Resize.args = {
+	referenceKey: "iframe",
+	validationTimeout: -1,
+	src: `${host}/iframe.html?viewMode=story&id=custom-iframe--resize-child`,
+};
+Resize.tags = ["!autodocs"];
+export const ResizeChild: StoryObj = {
+	render: () => <ChildResize />,
+	tags: ["!autodocs", "!dev"],
+};
+
+export const DetectSubmission = DefaultStoryTemplate<IIframeSchema>("iframe-submission").bind({});
+DetectSubmission.args = {
+	referenceKey: "iframe",
+	src: `${host}/iframe.html?viewMode=story&id=custom-iframe--detect-submission-child`,
+};
+DetectSubmission.tags = ["!autodocs"];
+export const DetectSubmissionChild: StoryObj = {
+	render: () => <ChildValidateSubmission />,
+	tags: ["!autodocs", "!dev"],
+};
+
+export const DefaultValue = DefaultStoryTemplate<IIframeSchema>("iframe-default-value").bind({});
+DefaultValue.args = {
+	referenceKey: "iframe",
+	validationTimeout: -1,
+	src: `${host}/iframe.html?viewMode=story&id=custom-iframe--default-child`,
+	defaultValues: "hello world",
+};
+DefaultValue.tags = ["!autodocs"];
+
+export const Reset = ResetStoryTemplate<IIframeSchema>("iframe-reset").bind({});
+Reset.args = {
+	referenceKey: "iframe",
+	validationTimeout: -1,
+	src: `${host}/iframe.html?viewMode=story&id=custom-iframe--default-child`,
+};
+Reset.tags = ["!autodocs"];
+
+export const ResetWithDefaultValues = ResetStoryTemplate<IIframeSchema>("iframe-reset").bind({});
+ResetWithDefaultValues.args = {
+	referenceKey: "iframe",
+	validationTimeout: -1,
+	src: `${host}/iframe.html?viewMode=story&id=custom-iframe--default-child`,
+	defaultValues: "hello world",
+};
+ResetWithDefaultValues.tags = ["!autodocs"];
+
+export const Overrides = OverrideStoryTemplate<IIframeSchema>("iframe-overrides").bind({});
+Overrides.args = {
+	referenceKey: "iframe",
+	validationTimeout: -1,
+	src: `${host}/iframe.html?viewMode=story&id=custom-iframe--validation-child`,
+	overrides: {
+		validationTimeout: 2000,
```

Comment:
this is the same value as the default and also the effect is not visible as the ChildValidation always validates immediately

how about the `title` attribute for the iframe or introducing a delay in the ChildValidation component?

Reply:
I decided to demo overriding of the styles instead, but anyway I think the overrides stories are unclear in general, thinking to have user trigger the overriding, I will look into changing them